nemo/collections/asr/models/ctc_models.py

```diff
@@ -211,6 +211,9 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         # Setup optional Optimization flags
         self.setup_optimization_flags()
 
+        # Adapter modules setup (from ASRAdapterModelMixin)
+        self.setup_adapters()
```

Comment:
Hmm not really, since there are speaker models and speech classification and speech SSL models all subclassing ASRModel. Safer to do it on a model basis.